SCAutolib/cli_commands.py

```diff
@@ -165,7 +169,47 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
+@click.command()
+@click.option('--init',
+              required=False,
+              default=False,
+              is_flag=True,
+              help="Initialize GUI for manual testing")
+@click.option("--assert-text",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Check if a word is found on the screen ")
+@click.option("--click-on",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Click on object containing word")
+@click.option("--check-home-screen",
+              required=False,
+              is_flag=True,
+              default=False,
+              show_default=True,
+              help="Check if screen appears to be the home screen")
+@click.pass_context
+def gui(ctx, init, assert_text, click_on, check_home_screen):
+    """ Run GUI actions manually """
+    from SCAutolib.models.gui import GUI
+    gui = GUI()
+    if init:
+        gui.__enter__()
```

Comment:
I think we also need to call __exit__ as well to finish the logging and stop gdm from running.